stubs/Pillow/PIL/Image.pyi

```diff
@@ -11,9 +11,13 @@ from ._imaging import (
     HUFFMAN_ONLY as HUFFMAN_ONLY,
     RLE as RLE,
 )
+from .ImageFilter import Filter
+from .ImagePalette import ImagePalette
 
 _Mode = Literal["1", "CMYK", "F", "HSV", "I", "L", "LAB", "P", "RGB", "RGBA", "RGBX", "YCbCr"]
 _Resample = Literal[0, 1, 2, 3, 4, 5]
+_Size = tuple[int, int]
+_Box = tuple[int, int, int, int]
```

Comment:
CI errors because mypy doesn't undestand lower-case `tuple` in type aliases yet.

```suggestion
_Size = Tuple[int, int]
_Box = Tuple[int, int, int, int]
```